packages/cubejs-cubestore-driver/src/CubeStoreDriver.ts

```diff
@@ -81,7 +81,15 @@ export class CubeStoreDriver extends BaseDriver implements DriverInterface {
   }
 
   public informationSchemaQuery() {
-    return `${super.informationSchemaQuery()} AND columns.table_schema = '${this.config.database}'`;
+    return `
+      SELECT columns.column_name as ${this.quoteIdentifier('column_name')},
+             columns.table_name as ${this.quoteIdentifier('table_name')},
+             columns.table_schema as ${this.quoteIdentifier('table_schema')},
+             columns.data_type as ${this.quoteIdentifier('data_type')}
+      FROM information_schema.columns as columns
+      WHERE columns.table_schema NOT IN ('information_schema', 'mysql', 'performance_schema', 'sys', 'INFORMATION_SCHEMA')
```

Comment:
this line may be redundant

Reply:
nope, it doesn't. The difference in this line: 
`FROM information_schema.columns as columns`
The cubestore doesn't make an alias automatically.

Reply:
Without this fixes we have an error 
![Screenshot 2023-03-20 at 14 37 43](https://user-images.githubusercontent.com/97888955/226535476-bcb9b0a8-9a24-45dc-aab9-c60fdb70de8c.png)